includes/admin/views/html-notice-regenerating-lookup-table.php

```diff
@@ -1,26 +1,33 @@
 <?php
 /**
- * Admin View: Notice - Regenerating thumbnails.
+ * Admin View: Notice - Regenerating product lookup table.
  *
  * @package WooCommerce/admin
  */
 
 defined( 'ABSPATH' ) || exit;
 
+$pending_actions_url = admin_url( 'admin.php?page=wc-status&tab=action-scheduler&s=wc_update_product_lookup_tables&status=pending' );
+$cron_disabled       = defined( 'DISABLE_WP_CRON' ) && DISABLE_WP_CRON;
 ?>
 <div id="message" class="updated woocommerce-message">
 	<p>
-		<strong><?php esc_html_e( 'WooCommerce is updating product data in the background.', 'woocommerce' ); ?></strong>&nbsp;
+		<strong><?php esc_html_e( 'WooCommerce is updating product data in the background.', 'woocommerce' ); ?></strong><br/>
 		<?php
-		echo wp_kses_post(
-			sprintf(
-				/* Translators: 1: Link, 2: link close. */
-				__( 'Some queries and reports may not show accurate results until this finishes. It will take a few minutes and this notice will disappear when complete.', 'woocommerce' ),
-				'<a href="' . esc_url( admin_url( 'admin.php?page=wc-status&tab=action-scheduler&s=wc_update_product_lookup_tables' ) ) . '">',
-				'</a>'
-			)
-		);
+		esc_html_e( 'Product display, sorting, and reports may not be accurate until this finishes. It will take a few minutes and this notice will disappear when complete.', 'woocommerce' );
+
+		if ( $cron_disabled ) {
+			echo '<br><span style="color:red;">' . esc_html__( 'Warning: WP CRON has been disabled on your install which may prevent this update from completing.', 'woocommerce' ) . '</span>';
```

Comment:
I wonder if for i18n it is better to combine this sentence with the one below. I'm saying this as WP.org i18n best practices recommend that we split at paragraphs and merge related sentences (https://codex.wordpress.org/I18n_for_WordPress_Developers#Best_Practices). Not sure if I'm understanding the recommendation correctly. The downside is that we will duplicate part of the string.

Reply:
They are related but I disagree they fit within the same sentence. Plus more importantly the second is a call to action, and merging them would require HTML to be inserted in to handle the link which is harder to translate. I vote to leave as-is.

Reply:
Sorry, my first comment wasn't clear. I wasn't suggesting that we combine the two sentences in a single sentence, but instead that we combine them in a single call to one of the translation functions. My understanding is that we should split calls to translation functions at paragraphs and not sentences and that is why I'm suggesting this. I agree that including HTML inside the translatable string is a con and so it is not clear which of the two approaches is the best. I'm happy to go with what you think is better in this case.